src/powershell-unix/powershell-unix.csproj

```diff
@@ -45,6 +45,11 @@
     <DefineConstants>$(DefineConstants);UNIX</DefineConstants>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(Configuration)' == 'LinuxDebug' ">
+    <DebugType>portable</DebugType>
```

Comment:
Yes, even for Windows the PDBs that are generated are portable. Hence, for code coverage we explicitly make the DebugType as full, which is Windows style full PDBs.